packages/experience-builder-sdk/src/types.ts

```diff
@@ -1,330 +1 @@
-    | 'left'
-    | 'right'
-    | 'top'
-    | 'bottom'
-    | 'left top'
-    | 'left center'
-    | 'left bottom'
-    | 'right top'
-    | 'right center'
-    | 'right bottom'
-    | 'center top'
-    | 'center center'
-    | 'center bottom';
-  cfHyperlink: string;
-  cfOpenInNewTab: boolean;
-};
-
-// We might need to replace this with Record<string, string | number> when we want to be React-agnostic
-export type CSSProperties = React.CSSProperties;
-
-export type ContainerStyleVariableName = keyof StyleProps;
-
-// cda types
-export type CompositionNode = {
-  definitionId: string;
-  children: Array<CompositionNode>;
-  variables: Record<string, CompositionComponentPropValue>;
-};
-
-export type CompositionDataSource = Record<string, DataSourceEntryValueType>;
-export type CompositionUnboundValues = Record<string, { value: CompositionVariableValueType }>;
-
-export type Breakpoint = {
-  id: string;
-  query: string;
-  displayName: string;
-  previewSize: string;
-};
-
-export type SchemaVersions = '2023-09-28' | '2023-06-27' | '2023-07-26' | '2023-08-23';
-
-export type Composition = {
-  title: string;
-  slug: string;
-  componentTree: {
-    breakpoints: Array<Breakpoint>;
-    children: Array<CompositionNode>;
-    schemaVersion: SchemaVersions;
-  };
-  dataSource: CompositionDataSource;
-  unboundValues: CompositionUnboundValues;
-};
-
-export type ExperienceEntry = {
-  sys: Entry['sys'];
-  fields: Composition;
-  metadata: Entry['metadata'];
-};
-
-export interface RawCoordinates {
-  left: number;
-  top: number;
-  width: number;
-  height: number;
-}
-
-export interface Coordinates extends RawCoordinates {
-  childrenCoordinates: RawCoordinates[];
-}
-
-export interface HoveredElement {
-  blockType: string | undefined;
-  nodeId: string | undefined;
-  blockId: string | undefined;
-}
-
-export interface DeprecatedExperienceStore {
-  composition: Composition | undefined;
-  entityStore: EntityStore | undefined;
-  isLoading: boolean;
-  children: Composition['componentTree']['children'];
-  breakpoints: Composition['componentTree']['breakpoints'];
-  dataSource: Composition['dataSource'];
-  unboundValues: Composition['unboundValues'];
-  schemaVersion: Composition['componentTree']['schemaVersion'] | undefined;
-  fetchBySlug: ({
-    experienceTypeId,
-    slug,
-    localeCode,
-  }: {
-    experienceTypeId: string;
-    slug: string;
-    localeCode: string;
-  }) => Promise<{ success: boolean; error?: Error }>;
-}
-
-export interface Experience {
-  entityStore?: EntityStore;
-  mode: InternalSDKMode;
-}
-
-/**
- * @deprecated please use `Experience` instead
- */
-export interface DeprecatedExperience {
-  /**
-   * @deprecated please don't use
-   */
-  client: ContentfulClientApi<undefined>;
-  /**
-   * @deprecated please don't use
-   */
-  experienceTypeId: string;
-  /**
-   * @deprecated please don't use
-   */
-  mode: InternalSDKMode;
-}
+export * from '@contentful/experience-builder-types/dist/src/types';
```

Comment:
should we just delete this `types.ts` file in general and import directly from types everywehre?

Reply:
I did this so we don't have to change it everywhere and then we can still have local types just for `ExperienceBuilder` which are not meant to be exported. So I think we should keep it.